civic_europe/assets/js/wagtail-videoembed-fix.js

```diff
@@ -0,0 +1,10 @@
+document.addEventListener('DOMContentLoaded', function() {
+  const videoEmbeds = document.querySelectorAll('.rich-text iframe')
+
+  for (const videoEmbed of videoEmbeds) {
+    const parentOfEmbed = videoEmbed.parentElement
+    // following classnames are bs4 classnames
+    parentOfEmbed.classList.add('embed-responsive')
+    parentOfEmbed.classList.add('embed-responsive-16by9')
+  }
+})
```

Comment:
why do we need this? now all video embeds had rich-tect class name any styling can be done https://github.com/liqd/a4-civic-europe/blob/cc3b6b33b96f79de4ffd7caeb042230d44dff81c/civic_europe/assets/scss/components/_blocks.scss#L245 along with extending any classes we want to add

Reply:
maybe it is obsolete, i added to mimic what we have here:

https://github.com/liqd/a4-opin/blob/da3766c759a031649466c74f07e696a8e4b5899d/euth_wagtail/assets/scss/wagtail-blocks/_blocks.scss#L31